hr_holidays_public/wizards/holidays_public_next_year_wizard.xml

```diff
@@ -9,44 +9,48 @@
-                            from 2012 and 2015 of the same country; '
-                            only the templates from 2015 will be taken into
-                            account).
+                        Use this wizard to create public holidays based on existing ones.
+                        If this is the first time you are creating a calendar,
+                        you should create it manually since this wizard takes
+                        previous years' calendars as templates.<br/>
+                    </div>
+                    <group>
+                        <group><field name="template_id" required="1" options="{'no_create': True}"/></group>
+                        <group>
+                            <field name="country_id"/>
+                            <field name="year"/>
+                        </group>
+                    </group>
+                    <div>
+                        Days to review are those that were marked with variable date
+                        in template's calendar. Here you can modify the date in case
+                        it is not the same than in the template.
                     </div>
-
                     <notebook>
-                        <page name="defaults" string="Defaults">
-                            <div>
-                            By default, the most recent public holidays
-                            for each country are used as template to create
-                            public holidays for the year following the templates.
-                            <br/><br/>
-                            Normally, you should not need to input anything in
-                            optional fields and only need to click on the button
-                            "Create".
-                            </div>
-                        </page>
-                        <page name="optional" string="Optional">
-                            <div>
-                            The below optional fields are here only to handle
-                            special situations like "2011 was a special year with
-                            an additional public holiday for the 150th
-                            anniversary of the Italian unification, so you want to
-                            replicate the 2010 Italian holidays to 2012."
-                            </div>
-                            <group>
-                                <field name="template_ids" />
-                                <field name="year" />
-                            </group>
+                        <page name="pending" string="Days to Review">
+                            <field name="pending_lines">
+                                <tree editable="bottom">
+                                    <field name="date"/>
+                                    <field name="name"/>
+                                    <field name="state_ids" widget="many2many_tags"/>
```

Comment:
`line_id` is missing and gives an error: You shoud add `<field name="line_id" invisible="1"/>`